lib/Sema/TypeCheckDistributed.h

```diff
@@ -20,35 +20,35 @@
 #include "swift/AST/ConcreteDeclRef.h"
 #include "swift/AST/DiagnosticEngine.h"
 #include "swift/AST/Type.h"
-#include <cassert>
 
 namespace swift {
 
 class AbstractFunctionDecl;
 class ConstructorDecl;
-class ActorIsolation;
-class AnyFunctionType;
 class ASTContext;
 class ClassDecl;
-class ClosureExpr;
-class ConcreteDeclRef;
-class CustomAttr;
 class Decl;
 class DeclContext;
-class EnumElementDecl;
 class Expr;
 class FuncDecl;
 class Initializer;
 class PatternBindingDecl;
 class ProtocolConformance;
-class TopLevelCodeDecl;
-class TypeBase;
 class ValueDecl;
```

Comment:
We should only need `Decl`, `ClassDecl`, and `ConstructorDecl` from this list.

Reply:
ok, cleaning up!